src/main/scala/com/fulcrumgenomics/illumina/SampleSheet.scala

```diff
@@ -168,12 +179,11 @@ object SampleSheet {
   */
 class SampleSheet(samples: Seq[Sample]) extends Iterable[Sample] {
 
-  // Validate some properties for a sample sheet
-  require(samples.map(_.sampleId).toSet.size == samples.length,
-    "Sample identifiers were not unique: " + samples.groupBy(_.sampleId).filter(_._2.length > 2).keys.mkString(", "))
-  require(samples.map(s => s"${s.sampleId} ${s.libraryId}").toSet.size == samples.length,
-    "Sample name and library identifier combinations were not unique: " +
-      samples.groupBy(s => s"${s.sampleId} ${s.libraryId}").filter(_._2.length > 2).keys.mkString(", "))
+  // Validate samples
+  samples.flatMap(_.lane).toSet[Int].toList.foreach { lane =>
```

Comment:
I'd suggest changing this to `.map(_.lane.getOrElse(1))` so you don't need the extra check on 186.